wrappers/wrapper.rb

```diff
@@ -449,6 +449,13 @@ def assert_no_setup_duration(vrp)
         }
     end
 
+    def assert_max_split_not_exceeded(vrp)
+      return true unless vrp.preprocessing_max_split_size
+
+      empties_or_fills = vrp.services.select{ |s| s.quantities.any?(&:fill) || s.quantities.any?(&:empty) }
+      (vrp.services.size - empties_or_fills.size) <= vrp.preprocessing_max_split_size
+    end
```

Comment:
Can you call `Interpreters::SplitClustering.split_solve_candidate?({ vrp: vrp })` instead of manual check (because the max_split conditions are more complicated then this (for exampl there is only one vehicle, even if vrp.size > max_split_size, we should be able to use Vroom (if other conditions are satisfed) because we can't do split_solve.

I didn't tested it but something like this should work:
```suggestion
    def assert_max_split_is_not_active(vrp) # in this case we can rename the assert as follows
    def assert_not_a_split_solve_candidate(vrp) # or like this..
      !Interpreters::SplitClustering.split_solve_candidate?({ vrp: vrp })
    end
```

Reply:
I thought we only want to limit on services. Are you saying VROOM is more efficient than ORtools for big instances that do not call split_solve ? The aim was initially to add a limit on number of services and we then decided to use max_split_size as limit value.

If we use `Interpreters::SplitClustering.split_solve_candidate?({ vrp: vrp })` instead of comparing with _max_split_size_ then we might still try to solve big problems with VROOM, I am not sure that is what we discussed.